examples/text_to_speech_synthesizer/waveglow_handler.py

```diff
@@ -62,6 +62,7 @@ def initialize(self, ctx):
         waveglow_config = waveglow_checkpoint['config']
         self.waveglow_model = WaveGlow(**waveglow_config)
         self.waveglow_model.load_state_dict(waveglow_state_dict)
+        self.waveglow_model = self.waveglow_model.remove_weightnorm(self.waveglow_model)
```

Comment:
@harshbafna Can you please explain the purpose of this?

Reply:
@chauhang : This has been taken up from example in : https://pytorch.org/hub/nvidia_deeplearningexamples_waveglow/